src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -227,32 +245,62 @@ export const Map: React.FC<PropsType> = ({
       )),
     [attendances, setIsRoomModalOpen, setSelectedRoom, venue]
   );
+
+  const zoomIn = useCallback(() => {
+    setZoom((zoom) => Math.min(zoom + ZOOM_INCREMENT, MAX_ZOOM));
+  }, []);
+
+  const zoomOut = useCallback(() => {
+    setZoom((zoom) => Math.max(zoom - ZOOM_INCREMENT, MIN_ZOOM));
+  }, []);
```

Comment:
Good use of state update pattern! I always forget that this method exists (which is the correct method to avoid using potentially stale data/race conditions/etc relating to updating the state, based on the existing state.